definitions/infra-awsapigatewayresource/definition.yml

```diff
@@ -0,0 +1,11 @@
+domain: INFRA
```

Comment:
For EP: This one is set to NOT_VISIBLE in our schema. Is one of those from beyond. we are still waiting an answer on what to do with them. should we allow this? Maybe reach to beyond again and take a final stand on this being visible or being deleted?

Reply:
+1, this is a public API so I'd avoid adding anything that is still not confirmed

Reply:
I would agree on reaching out again, we can point them to this PR to explain that the decision is a blocker?

Reply:
easier to be conservative now rather than later.. :) has anyone taken to documenting approaches so we can fully decide what to do here?